.github/workflows/workflow.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        xcode: [ "${{ env.XCODE_VERSION }}" ]
+        xcode: [ '13.1' ]
```

Comment:
@CAMOBAP assume unintentional

Reply:
Nope, unfortunately, `env` referencing doesn't work in the matrix. I tried but it doesn't work: https://github.com/hCaptcha/HCaptcha-ios-sdk/actions/runs/3986217990